gprofiler/profilers/python_ebpf.py

```diff
@@ -157,7 +157,7 @@ def test(self) -> None:
             "--duration",
             "1",
         ] + self._offset_args()
-        process = start_process(cmd, via_staticx=True)
+        process = start_process(cmd, via_staticx=False)
```

Comment:
This is the last site using `via_staticx=True`, let's make it a default parameter with `False` value and remove the `via_staticx=False` everywhere.
I do suggest we keep support for `via_staticx` in case we need it in the future.

Reply:
👍 
Will do.